plugins/scanners/scanoss/src/main/kotlin/ScanOssConfig.kt

```diff
@@ -46,7 +45,8 @@ data class ScanOssConfig(
         const val API_KEY_PROPERTY = "apiKey"
 
         fun create(options: Options, secrets: Options): ScanOssConfig {
-            val apiUrl = options[API_URL_PROPERTY] ?: ScanOssService.DEFAULT_API_URL
+            // TODO: Hard-code the default URL until https://github.com/scanoss/scanoss.java/issues/11 is resolved.
```

Comment:
Not worded like a TODO item, e.g. "Remove hard-coded default URL once ... is resolved."